docs/cloud/integrations/apachesuperset.md

```diff
@@ -0,0 +1,122 @@
+
+Now add a `Dataset` which is a representation of a table you want exposed for querying. To do that click on `+Dataset` from the Datasets tab and update the database, schema and table details. Here we add a dataset using the `azure_compute_disk` table to list the Azure compute disks.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"50%"}}>
+<img src="/images/docs/cloud/apache-superset-dataset-creation.png" />
+</div>
+
+To create the table, select the `azure_compute_disk` dataset, set the `QUERY MODE` to `RAW RECORDS`, update the columns to be displayed and click `CREATE CHART`. Superset will then preview the data.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/apache-superset-disk-preview.png" />
+</div>
+
+## Create charts with custom queries.
+
+You can augment your SQL queries through Supersets `SQL Lab` under the SQL tab and visualize it with a suite of charts. To try that, click `SQL Lab` and paste this SQL that returns the number of compute disks region wise.
+
+```
+select
+  region,
+  count(name)
+from
+  azure_compute_disk
+group by
+  region;
+  ```
+
+Once the result is returned, click `CREATE CHART` and select `PIE CHART`. Now Update the region and count columns into the dimensions field, set the metric field to aggregate `MAX` count and click `UPDATE CHART`. Custom queries can also be saved as datasets.
+
+<div style={{"marginTop":"1em", "marginBottom":"1em", "width":"90%"}}>
+<img src="/images/docs/cloud/apache-superset-custom-query.png" />
+</div>
+
+## Create dashboard to analyze resources
+
+Charts that display data are the building blocks of Apache Superset dashboards. We'll focus here on a creating a dashboard that monitors the Azure VMs and disks. To being, click `+Dashboard` from the Dashboards tab, and click `+Create a New Chart`. Then Create datasets and charts with `azure_compute_virtual_machine_metric_cpu_utilization_hourly` and `azure_compute_disk_metric_read_ops_daily` tables. Finally paste these queries to create dataset charts for VMs with disks not encrypted by customer managed key and list of unattached disks.
+
+```
+select
+  count(disk.encryption_type)
+from
+  azure_compute_disk as disk
+  join azure_compute_virtual_machine as vm on disk.name = vm.os_disk_name
+where
+  not disk.encryption_type = 'EncryptionAtRestWithCustomerKey'
+  ```
+
+```
+select
+  name,
+  disk_state,
+  sku_tier,
+  time_created,
+  encryption_type,
+  network_access_policy
+from
+  azure_compute_disk
+where
+  disk_state = 'Unattached';
+  ```
+
+Superset provides a variety of chart types. Here we use `Bar Chart` for azure_compute_virtual_machine_metric_cpu_utilization_hourly, `Funnel Chart` for azure_compute_disk_metric_read_ops_daily, `Table` for list of unattached disks and `Gauge Chart` for VMs with disks not encrypted by customer managed key. The dashboard can also be saved as an image by simply clicking on `Download as image` from the dashboard menu.
```

Comment:
```suggestion
Superset provides a variety of chart types. Here we use `Bar Chart` for `azure_compute_virtual_machine_metric_cpu_utilization_hourly`, `Funnel Chart` for `azure_compute_disk_metric_read_ops_daily`, `Table` for list of unattached disks and `Gauge Chart` for VMs with disks not encrypted by customer-managed key. The dashboard can also be saved as an image by clicking on `Download as image` from the dashboard menu.
```

Reply:
Nice mix of charts!

Reply:
Thank you. Wanted to focus on just one service but multiple features